frame/support/procedural/src/construct_runtime/parse.rs

```diff
@@ -393,6 +395,8 @@ impl Parse for PalletPartKeyword {
 			Ok(Self::Storage(input.parse()?))
 		} else if lookahead.peek(keyword::Event) {
 			Ok(Self::Event(input.parse()?))
+		} else if lookahead.peek(keyword::Error) {
```

Comment:
Runtimes that still declare these parts manually, will forget to add the error type and then it will not be exported properly...

Reply:
I might not have all the context here. Is this only affecting the `pallet::error` macro? Is there a safer way of fetching this info? 🤔